mobile_app_connector/models/mobile_writing.py

```diff
@@ -109,7 +109,7 @@ def get_json(self):
             + "/"
         return {
             'templates': [
-                {'templateURL': base_url + str(t.id) + "/template_image",
+                {'templateURL': "http://" + base_url + str(t.id) + "/template_image",
```

Comment:
The `web.external.url` parameter should include the 'http://' prefix. I think it's best to remove it from here and correcting your parameter in your database. In production we have https:// as well

Reply:
I didn't know...